sample_consensus/include/pcl/sample_consensus/impl/sac_model_ellipse3d.hpp

```diff
@@ -0,0 +1,894 @@
+/*
```

Comment:
Please use the short SPDX license text: https://pcl.readthedocs.io/projects/tutorials/en/master/writing_new_classes.html#licenses (in all new files)

Reply:
Done!